common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -100,6 +100,26 @@ class InheritanceMixin(XBlockMixin):
         scope=Scope.settings,
         default="finished",
     )
+
+    show_correctness = String(
```

Comment:
Thanks for the heads up, @e-kolpakov.  I don't know what the usual method for passing settings down from subsections is supposed to be, so was just working from the examples in the code (`due`, `graceperiod`, `showanswer`).

It was also nice to let the default `submitted_message` be part of the Advanced Settings, and not have to include that in the subsection settings.